src/js/user-profile/actions/index.js

```diff
@@ -72,5 +73,7 @@ export function acceptTerms(termsName) {
       }),
       () => dispatch({ type: ACCEPTING_LATEST_MHV_TERMS_FAILURE })
     );
+
+    getUserData(dispatch);
```

Comment:
Do you intend to call this synchronously? If you want to `getUserData` in response to the POST request (which I'm assuming is what you wanted to do), you'll want to add it to the handlers.

Like instead of:

```
() => dispatch({ type: ACCEPTING_LATEST_MHV_TERMS_SUCCESS }),
```

Perhaps you might want:

```
() => {
  dispatch({ ... });
  getUserData(dispatch);
},
```

Otherwise, you're doing the fetch from `apiRequest` and then immediately calling `getuserData`. The completion of the request could occur after the `getUserData` since it's asynchronous.